includes/REST_API/Stories_Base_Controller.php

```diff
@@ -81,7 +91,7 @@ protected function prepare_item_for_database( $request ) {
 		}
 
 		if ( isset( $request['content'] ) ) {
-			$prepared_post->post_content = $this->base64_decode( $prepared_post->post_content );
+			$prepared_post->post_content = ( new Decoder( $this->experiments ) )->base64_decode( $prepared_post->post_content );
```

Comment:
If we add this to the `__construct`

```
$this->decoder = new Decoder( $this->experiments );
```

Meaning this line would look like this. 

```
$prepared_post->post_content = $this->decoder->base64_decode( $prepared_post->post_content );
```

Reply:
Sure we can do that